main.js

```diff
@@ -135,5 +135,22 @@ function removeCard(event) {
     var index = outfits.findIndex(outfit => outfit.id === outfitId);
     outfits.splice(index, 1);
     addOutfitsToLocalStorage();
+  } else {
+    var outfitId = event.target.closest('.saved_outfit').id;
+    loadSavedOutfit(outfitId);
+  }
+}
+
+function loadSavedOutfit(id){
+  clearBearDisplay();
+  for(var i =0; i<outfits.length; i++){
+    if(outfits[i].id==id){
+      currentOutfit = outfits[i];
```

Comment:
If you save an outfit, reload the page, then load the outfit, you can no longer edit the current outfit, I think this line right here might be causing the problem

Reply:
I'm getting an error saying that currentOutfit.addGarment isn't a function, so it seems like you're overwriting currentOutfit somehow

Reply:
Fixed that. Good catch.

Reply:
used Object.assign to assign all values from one saved object to the current object.